src/cljs/org/broadinstitute/firecloud_ui/page/methods_configs_acl.cljs

```diff
@@ -60,19 +115,33 @@
                                                          (:background-gray style/colors))}
                               :disabled (< i (:count-orig @state))
                               :spellCheck false
-                              :defaultValue (acl-entry "user")})
+                              :defaultValue (:user acl-entry)
+                              :onChange (fn [e]
+                                          (let [new-val (-> e .-target .-value)
+                                                new-val-is-public (= "public" new-val)]
+                                            (when new-val-is-public
+                                              (do
+                                                (js/alert "Cannot set value to 'public'!  Use the check-box instead.")
+                                                (set! (-> e .-target .-value) "")))))})
```

Comment:
As with the previous PR, don't do this.  Validate the text fields when the OK button is pressed.  What if someone's ID starts with "public"?